crates/forge/bin/cmd/test/mod.rs

```diff
@@ -397,13 +397,6 @@ impl TestOutcome {
             }
             println!();
         }
-
-        let successes = self.successes().count();
-        println!(
-            "Encountered a total of {} failing tests, {} tests succeeded",
-            Paint::red(failures.to_string()),
-            Paint::green(successes.to_string())
-        );
```

Comment:
let's re-add this

Reply:
Added here: https://github.com/foundry-rs/foundry/pull/5731/commits/e409e275015c65dd669cc835174c1ef4a7199389